scripts/experiments_tabular/aae_full.jl

```diff
@@ -122,8 +144,18 @@ if abspath(PROGRAM_FILE) == @__FILE__
 	try_counter = 0
 	max_tries = 10*max_seed
 	cont_string = (contamination == 0.0) ? "" : "_contamination-$contamination"
+	sampling_string = sampling == "bayes" ? "_bayes" : "" 
+	prefix = "experiments$(sampling_string)/tabular$(cont_string)"
+	dataset_folder = datadir("$(prefix)/$(modelname)/$(dataset)")
 	while try_counter < max_tries
-	    parameters = sample_params()
+	    if sampling == "bayes"
+			parameters = GenerativeAD.bayes_params(
+									create_space(), 
+									dataset_folder,
+									sample_params; add_model_seed=true)
+		else
+			parameters = sample_params()
+		end
 
 	    for seed in 1:max_seed
 			savepath = datadir("experiments/tabular$cont_string/$(modelname)/$(dataset)/seed=$(seed)")
```

Comment:
Change this to savepath = joinpath(dataset_folder, "seed=$(seed)"). Currently the models are stored under experiments instead of experiments_bayes.